test/app.createRouter.js

```diff
@@ -0,0 +1,30 @@
+var express = require('../')
+var assert = require('assert')
+
+describe('app.createRouter()', function () {
+
+  it('should create a router that inherits settings from app.settings', function () {
+    var app = express()
+    app.set('strict routing', true)
+    app.set('case sensitive routing', true)
+
+    var router = app.createRouter()
+
+    assert(router.strict, 'Router did not inherit strict option from app.settings')
+    assert(router.caseSensitive, 'Router did not inherit case sensitive option from app.settings')
+  })
+
+  it('should allow options param to override app.settings when creating router', function () {
+    var app = express()
+    app.set('strict routing', true)
+    app.set('case sensitive routing', false)
+
+    var router = app.createRouter({
+      strict: false,
+      caseSensitive: false
+    })
+
+    assert.equal(router.strict, false, 'options param strict did not override app.settings')
+    assert.equal(router.caseSensitive, false, 'options param caseSensitive did not override app.settings')
```

Comment:
How do you know if this false is coming from the app settings or from the override?

Reply:
It's false by accident, I'll get it fixed.